arch/x86/Kconfig

```diff
@@ -72,14 +72,25 @@ config X86_IAMCU
 	assembly code will need to be updated to conform to the new calling
 	convention.  If in doubt say N
 
+menu "Memory Management"
 config X86_MMU
 	bool
 	default n
-	prompt "Memory Management Unit"
+	prompt "Enable Memory Management Unit"
 	help
 	This options enables the memory management unit present in x86. Enabling
 	this will create boot time page table structure.
 
+config X86_PAE_MODE
+	bool
+	depends on X86_MMU
+	default n
+	prompt "Enable PAE page tables"
+	help
+	When selected the PAE mode is enabled. The boot time page tables will be
```

Comment:
I would add some language that the practical use for this is to enable specifying the "No Execute" bit for pages, at the expense of greater RAM consumption for the page tables.